### ISSUE TITLE
Define 

### ISSUE THREAD

**Issue by mmmavis** [mozilla/mozfest-event-app#329]

**Comment by edrushka** [comment]
Can you clarify, what are the other labels in addition to 'everyone'? I'm not sure what this category this label represents.

**Comment by Saallen** [comment]
Hi @mmmavis 
I would be against marking D&D as everyone- as an event all should attend- these are just part of the program as are all sessions

**Comment by xmatthewx** [comment]
If I remember correctly, certain sessions were marked as important last year, or relevant to everyone. In the app, they were simply highlighted with a red line. These included main stage, science fair, maybe lunch, etc. 

I'm unsure about the word "everyone" appearing in the app. I'd wager it'll confuse more than a few people. Is there a way we can highlight this for attention without a word label?

**Comment by edrushka** [comment]
+1 to MW's comment. Everyone is welcome to attend _everything_, so this label might cause confusion.

**Comment by kristinashu** [comment]
Would there be a different word we could use? I'm not sure a color bar or an icon would be able to convey what we want.

**Comment by edrushka** [comment]
I'm still unclear on how a label like this would be used. Are there other categories of people? What are the other groupings in addition to 'everyone'?

**Comment by mmmavis** [comment]
@edrushka I agree "everyone" label can be confusing and the wording should be changed. We initially thought that the app should have a way to highlight registration/plenaries/lunch/science fair/... types of events so they stand out more from the hundreds of sessions. "everyone" was the word we gave this category of events but we can definitely change the wording to something else.

**Comment by edrushka** [comment]
Oh, I see - this indicates programmatic pieces. Maybe "program" or "MozFest" or "schedule"? I'm having trouble offering suggestions without seeing how this word will be used in context.

**Comment by xmatthewx** [comment]
I propose that we just stick a yellow or pink icon at the end of the title. Something small so it doesn't look like you should tap it. I think that any word we pick will be more confusing than helpful. 

The schedule will function fine with no indicator, and we shouldn't commit too much more time to this issue.

**Comment by edrushka** [comment]
+1 Perhaps we could use the little gear. That's a good visual indicator of the overall festival.

**Comment by kristinashu** [comment]
Since there's already a lot of icons, I'm hesitant to add in another one. I guess people don't need to know why a session is highlighted, just that it is highlighted somehow. SO after extensive thought from @mmmavis @flukeout @sabrinang and I, we think a pink vertical bar is the way to go. 

<img width="322" alt="screen shot 2016-09-19 at 5 16 26 pm" src="https://cloud.githubusercontent.com/assets/1068783/18653228/5a4a3032-7e8d-11e6-8e52-a7d9cc840955.png">

**Comment by mmmavis** [comment]
To summarize, 

The event app spreadsheet will have a sheet(tab) called "programmatic pieces" (we can fix the wording). This is the sheet that we put registration/plenaries/lunch/science fair/... types of events. Only sessions listed in this sheet will be highlighted by a pink vertical bar on the app. 🎉

**Comment by xmatthewx** [comment]
Perfect. It's a nice iteration on the horizontal red bar we used last year. 😉
